src/unarchiving/unarchiver.e

```diff
@@ -34,29 +33,31 @@ feature {NONE} -- Initialization
 			no_unarchiving_in_progress: unarchiving_finished
 		end
 
-feature -- Status
+feature -- Access
 
 	name: STRING_8
-			-- Name that will be used when printing error messages this unarchiver reports
-
-	unarchiving_finished: BOOLEAN
-			-- Flag that indicates whether unarchiving finished
-		do
-			Result := has_error or attached active_header implies unarchived_blocks = required_blocks
-		end
+			-- Name used when printing error messages reported by Current unarchiver.
 
 	active_header: detachable TAR_HEADER
-			-- The header for which unarchiving is in progress
+			-- Current header during unarchiving.
```

Comment:
take the habit to end a comments with a point "." 

this way, we know the comment was not truncated.